### ISSUE TITLE
[Warning and Notice] Fix Warnings and Notices 

### ISSUE THREAD

**Issue by kjnanda** [rtCamp/rtMedia#1647]
- [ ] **1. When the user post the update with media on activity page, it shows below notices in the console** 

`This is present in Verison - 4.6.2 and 4.6.3 both`

![debug_log upload from activity](https://user-images.githubusercontent.com/15173772/81035892-7cf95e00-8eba-11ea-87da-246d772d7532.jpg)

- [ ] **2. When the user comments on uploaded media from the lightbox, it shows below notice and warnings**

`This is present in Verison -  4.6.3`

![debug_log](https://user-images.githubusercontent.com/15173772/81035993-f3965b80-8eba-11ea-9187-e855fcb33b15.jpg)

`2576339 ` `2578864` `2586313`

**Comment by pooja-muchandikar** [comment]
@SID177 

The above warnings and notices are not shown but On uploading audio file following warning is shown


```
PHP Notice:  Trying to access array offset on value of type null in /var/www/htdocs/wp-content/plugins/rtMedia/app/main/controllers/template/rtmedia-functions.php on line 3163
```

Please check on this.

**Comment by pooja-muchandikar** [comment]
Tested with `pre-release` branch and the warning/error mentioned in the comment [here](https://github.com/rtCamp/rtMedia/issues/1647#issuecomment-653025519)  are fixed.

**Comment by pooja-muchandikar** [comment]
Tested with `develop` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Released with`v4.6.4`
